distributed/dashboard/components/scheduler.py

```diff
@@ -4326,16 +4326,12 @@ def __init__(self, scheduler, **kwargs):
                 "comm_memory": [],
                 "comm_memory_limit": [],
                 "comm_buckets": [],
-                "comm_avg_duration": [],
-                "comm_avg_size": [],
```

Comment:
We're losing a little bit of functionality here.
IMHO it's not a big deal. Worth noting that we still have the information under the fine performance metrics (you'll have to calculate seconds/count and bytes/count yourself).